ipn/ipnlocal/network-lock.go

```diff
@@ -111,6 +114,13 @@ func (b *LocalBackend) tkaFilterNetmapLocked(nm *netmap.NetworkMap) {
 	} else {
 		b.tka.filtered = nil
 	}
+
+	// Check that we ourselves are not locked out, report a health issue if so.
+	if nm.SelfNode != nil && b.tka.authority.NodeKeyAuthorized(nm.SelfNode.Key, nm.SelfNode.KeySignature) != nil {
+		health.SetTKAHealth(errors.New("this node is locked out; it will not have connectivity till it is signed"))
```

Comment:
until

Reply:
Oooooh, is that a thing we can do / do we do that for anything else? I like the idea of a specialized "uwu oopsie daisies y'all locked out, do the thing" page.

Reply:
yeah `git grep tailscale.com/s/`